aws-rds-dbclusterparametergroup/src/test/java/software/amazon/rds/dbclusterparametergroup/CreateHandlerTest.java

```diff
@@ -405,6 +440,77 @@ public void handleRequest_SuccessWithParameters() {
+     * expects a maximum of 20 parameters that can be modified in a single request.
+     * (https://docs.aws.amazon.com/cli/latest/reference/rds/modify-db-parameter-group.html)
+     *
+     * In the case that the parameters included in the CFN template exceed the request limit, we could end up in
+     * a condition in which 2 related parameters existing in the CFN template, end up in 2 different requests.
+     * We need to ensure that all related parameters are sent in the same request as defined
+     * in BaseHandlerStd.PARAMETER_DEPENDENCIES
+     *
+     * This test ensure that "aurora_enhanced_binlog" and "binlog_backup" get bundled in the same request after
+     * the split logic that happens in BaseHandlerStd.modifyParameters
+     */
+    @Test
+    public void handleRequest_SuccessSplitParameters() {
+        when(rdsClient.createDBClusterParameterGroup(any(CreateDbClusterParameterGroupRequest.class)))
+                .thenReturn(CreateDbClusterParameterGroupResponse.builder()
+                        .dbClusterParameterGroup(DB_CLUSTER_PARAMETER_GROUP)
+                        .build()
+                );
+
+        when(rdsClient.modifyDBClusterParameterGroup(any(ModifyDbClusterParameterGroupRequest.class)))
+                .thenReturn(ModifyDbClusterParameterGroupResponse.builder().build());
+
+        when(rdsClient.describeDBClusters(any(DescribeDbClustersRequest.class)))
+                .thenReturn(DescribeDbClustersResponse.builder()
+                        .dbClusters(DBCluster.builder().dbClusterParameterGroup("group").status("available").build())
+                        .build());
+
+        final DBClusterParameterGroup dbClusterParameterGroup = DBClusterParameterGroup.builder()
+                .dbClusterParameterGroupArn(ARN)
+                .dbClusterParameterGroupName(RESOURCE_MODEL.getDBClusterParameterGroupName())
+                .dbParameterGroupFamily(RESOURCE_MODEL.getFamily())
+                .description(RESOURCE_MODEL.getDescription()).build();
+
+        when(rdsProxy.client().listTagsForResource(any(ListTagsForResourceRequest.class)))
+                .thenReturn(ListTagsForResourceResponse.builder()
+                        .tagList(Tag.builder().key(TAG_KEY).value(TAG_VALUE).build())
+                        .build());
+
+        // Defining 2 parameters that have to be bundled in the same request. We do this by including them in an
+        // insert-ordered Map, and being far away one from each other. With these conditions, they would naturally
+        // be bundled in 2 different API requests to "modify dbclusterparametergroup".
+        ResourceModel resourceModel = RESOURCE_MODEL.toBuilder()
+                .parameters(InsertionOrderedParamBuilder.builder()
+                        .addParameter("aurora_enhanced_binlog", "1")
+                        .addRandomParameters(BaseHandlerStd.MAX_PARAMETERS_PER_REQUEST)
```

Comment:
would be good to add hardcoded parameters that should not be bundle and that they existed in 2 separate requests.